linter.py

```diff
@@ -25,13 +26,13 @@ class Flow(Linter):
     version_requirement = '>= 0.1.0'
     regex = r'''(?xi)
         # Warning location and optional title for the message
-        ^.+/(?P<file_name>[^/]+\.(js|html)):(?P<line>\d+):(?P<col>\d+),\d+:\s*(?P<message_title>.+)$\r?\n
+        ^.+\/(?P<file_name_1>.+):(?P<col_1>(\d+:\d+,(\d+:)?\d+)):\s(?P<message_title>.+)?\r?\n
```

Comment:
This is the first time i've written code in python and i've only come in contact with regex one or two times earlier, so i've used the tool http://www.regexr.com/ to validate the strings. I understand that it isn't the most optimal solution, but from what i've tested, it works.

I've now opened issues on the problems i've found.